src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -1,74 +1,67 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <!-- 
```

Comment:
Reviewers: to help understand this change... The old way was a CodeTaskFactory that scanned an item collection and pulled out metadata and wrote it to a file.  It was expected TestProperties would grow and that this would be easier to maintain.  But TestProperties is relatively stable, and it became far easier to just replace all that complexity with a CDATA section that would expand MsBuild properties and write to disk.  Future additions to TestProperties will require updating this CDATA section.